docs/ctf-bots.md

```diff
@@ -0,0 +1,35 @@
+# CTF Q-bots
+
+Source code of bots was miss, for now we only have [binaries](https://github.com/airmash-refugees/Q-bots)
+
+## Commands
+
+### Common
+
+* `#status` - show amount of bots in your team
+* `#help` - show help for bots commands
+* `#help <command without #>` to see more details about specified command
+
+### Leadership
+
+Bots choose leader inside a team, who can give commands to bot.
+
+* If current leader is AFK you can start elections with:
+    * send `#status` command
+    * in next 10 seconds send `/elections` command
+
+* If you are leader and want to pass leadership to another player: `#leader <NICKNAME>`
+
```

Comment:
There is also [`/usurp`](https://github.com/wight-airmash/ab-server/blob/master/docs/commands.md#usurp) command in case the leader does something strange or nothing.

Reply:
What does this command do?

Reply:
Allows you to take the leader position "without any questions" if the leader has fewer points. Q-bots start the re-election every 10 minutes. Their algorithm prefers to select players with fewer points who leave a #yes message, thus trying to give newcomers a chance to control the bots. The idea is interesting, but in practice the result is not good. The command gives a strong player a chance to get back the control. Of course, bounty is very approximate about the strength of the player, but this is the most understandable parameter for everyone.

Reply:
Updated